diagnostic.md

```diff
@@ -23,7 +26,7 @@ Write the jQuery code to navigate from one of the list item (`li`) DOM nodes to
 the `form` DOM node. (Assume the list item has a class of "listerItem").
 
 <!-- your answer starts here -->
-
+listerItem = $('li')
```

Comment:
This will set a variable (if strict mode is off) named `listerItem` to a jQuery object containing all matched `<li>` elements.  A few possible solutions are `$('li').parent().parent().find('form')`, `$('li').parent().parent().children('form')`, and `$('.listerItem').parent().parent().$('form')`.